pkg/build/strategies/sti/sti.go

```diff
@@ -567,7 +567,7 @@ func (b *STI) Execute(command string, user string, config *api.Config) error {
 			if err != nil {
 				// we're ignoring ErrClosedPipe, as this is information
 				// the docker container ended streaming logs
-				if glog.V(2) && err != io.ErrClosedPipe {
+				if glog.V(2) && err != io.ErrClosedPipe && err != io.EOF {
```

Comment:
Why are we adding the new condition?

Do you know why, here and in `layered.go`, we have `glog.V(2)` in the if clause instead of just writing `glog.V(2).Errorf(...)`?